openapi.json

```diff
@@ -2064,7 +2064,7 @@
           "enum": [
             "HOT",
             "WARM",
-            "COLD",
+            "INACTIVE",
             "FROZEN"
```

Comment:
**suggestion:**  with the new renaming, shall it be  

```
"ACTIVE",
"INACTIVE",
"OFFLOADED"
```

Reply:
Thanks! I will delete this file actually as we use the OpenAPI spec from the main `weaviate/weaviate` repo these days